compiler/rustc_middle/src/ty/consts/valtree.rs

```diff
@@ -23,8 +23,8 @@ pub enum ValTree<'tcx> {
     /// of these types have the same representation.
     Leaf(ScalarInt),
 
-    //SliceOrStr(ValSlice<'tcx>),
-    // dont use SliceOrStr for now
```

Comment:
this is not a typo, this is commented out code.

Reply:
Sorry, fixed it now

Reply:
still not fixed, `// dont use SliceOrStr for now` is also part of the comment and not documentation

Reply:
I have undone everything besides changing `// dont ` to `// don't`. Please have a look.